### PR TITLE
devex - fixing an issue with the clear-env.sh script to be more use friendly

### PR DESCRIPTION
**Pull request by codyseibert**

### CODE REVIEW

File: clear-env.sh [hunk 1]
```diff
@@ -12,15 +12,21 @@
   "DEFAULT_ACCOUNT_PASS" \
   "ENV" \
   "USTC_ADMIN_USER" \
-  "DEPLOYING_COLOR"
+  "EFCMS_DOMAIN"
```
Comment (mrinsin) [inline clear-env.sh:15]: Is this being used in the script? Cant find it but maybe I missed it somewhere

**Review by mrinsin — APPROVED** [review]

File: clear-env.sh (hunk 1 above)
Comment (codyseibert) [reply clear-env.sh:15]: it's used in one of the nested node scripts.

Comment (codyseibert) [reply clear-env.sh:15]: I think the import judge one

Comment (mrinsin) [reply clear-env.sh:15]: Cool, thanks!

**Pull request merged**